airbyte-integrations/connectors/destination-elasticsearch/metadata.yaml

```diff
@@ -2,7 +2,7 @@ data:
   connectorSubtype: api
   connectorType: destination
   definitionId: 68f351a7-2745-4bef-ad7f-996b8e51bb8c
-  dockerImageTag: 0.1.6
+  dockerImageTag: 0.1.7
```

Comment:
There should be a similar file to update in the destination-elasticseach-strict-encrypt directory as well